numpy/lib/twodim_base.py

```diff
@@ -361,6 +380,16 @@ def tri(N, M=None, k=0, dtype=float):
         and `k` > 0 is above.  The default is 0.
     dtype : dtype, optional
         Data type of the returned array.  The default is float.
+    like : array_like
+        Reference array-like object created by a downstream library, such
+        as CuPy, Dask, xarray, etc. If specified, the function will be
```

Comment:
Same here.

Reply:
Sorry, resolving all of these doc duplicates, I hope thats fine. The PR seemed to start having an unwieldy amount of comments...